donkeycar/parts/controller.py

```diff
@@ -518,18 +518,21 @@ def __init__(self, *args, **kwargs):
 
         self.axis_names = {
             0x00: 'left_stick_horz',
-            0x02: 'left_trigger',
-            0x05: 'right_trigger'
+            0x5: 'right_stick_horz',
+            0x1: 'left_stick_vert',
+            0x4: 'left_stick_vert',
+            0xa: 'left_trigger',
+            0x9: 'right_trigger'
         }
 
         self.button_names = {
             0x130: 'a_button',
             0x131: 'b_button',
-            0x133: 'x_button',
```

Comment:
Are you sure these new names and axes work on a Jetson ?
This is a load bearing part of Donkey, and I want to be extra careful.